llvm/tools/opt/lib/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# We don't want to link this into libLLVM
```

Comment:
for my understanding, how do we tell the build system not to link this into libLLVM?